package/librtcm/librtcm.mk

```diff
@@ -4,9 +4,10 @@
 #
 ################################################################################
 
-LIBRTCM_VERSION = v0.2.41
-LIBRTCM_SITE = https://github.com/swift-nav/librtcm
+LIBRTCM_VERSION = 8f6c945f137a572716f7a4a2a75797260ac1e280
```

Comment:
let's do this properly and cut a release. there's no rush to fix this and unnecessary tech debt doesn't seem like the right call